### PR TITLE
Black/2021 05 18/node tests

### PR DESCRIPTION
**Pull request by MichaelEdwardBlack**
Passing integration tests for the wallet and provider services
image

The provider service InvitationStatus test does not actually run because it is not implemented on the backend yet

### CODE REVIEW

File: .github/workflows/node.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      run: 
+        working-directory: ./node
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with: 
+          token: ${{ secrets.ACCESS_TOKEN }}
+          node-version: ${{ matrix.node-version }}
+        uses: actions/checkout@v1
```
Comment (tmarkovski) [inline .github/workflows/node.yml]: May be missing a `-` here as in `- uses: actions/checkout@v1`

File: node/package.json [hunk 2]
```diff
@@ -3,22 +3,27 @@
...
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@grpc/grpc-js": "^1.2.8",
-    "google-protobuf": "^3.15.1"
+    "@trinsic/okapi": "^1.0.0-preview.10",
+    "google-protobuf": "^3.17.0",
+    "grpc": "^1.24.4",
+    "pb-util": "^0.1.3"
```
Comment (tmarkovski) [inline node/package.json]: Is this package used?

**Review by tmarkovski — APPROVED** [review]

File: node/package.json (hunk 2 above)
Comment (MichaelEdwardBlack) [reply node/package.json]: nope, it's not. I'm removing it now

**Comment by tmarkovski** [comment]
Fixes https://github.com/trinsic-id/server/issues/38

**Pull request merged**